src/utils/verbose.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#ifndef VERBOSE_HPP
+#define VERBOSE_HPP
+
+#include <cinttypes>
+#include <mutex>
+#include <stdio.h>
+#include <sys/time.h>
+
+static double get_msec() {
+    struct timeval time;
+    gettimeofday(&time, nullptr);
```

Comment:
#include <chrono>

auto tag_0 = std::chrono::high_resolution_clock::now();
decltype(tag_0) tag_1;
static std::chrono::duration<double> diff_0_1 = tag_0 - tag_0;
...
tag_1 = std::chrono::high_resolution_clock::now();
diff_0_1 += tag_1 - tag_0;

Reply:
need to use class